### PR TITLE
Check number of items not the relation

### PR DESCRIPTION
**Pull request by smeeckaert**
There is a bug that wrongly set the number of items when the has many is populated by an array instead of a relationship.

This cause some issues determining the index of new elements.

Also i set by default this number of items at 0 if the default_item directive is set to false.

### CODE REVIEW

**Comment by shaoshiva** [comment]
:+1:

File: views/hasmany/items.view.php [hunk 1]
```diff
@@ -1,24 +1,26 @@
 <?php
 \Nos\I18n::current_dictionary('novius_renderers::default');
+$listItems = null;
+if (!empty($relation) && !empty($item)) {
+    $listItems = $item->{$relation};
+}
+elseif ($value) {
```
Comment (jguyomard) [inline views/hasmany/items.view.php]: "} elseif (!empty($value)) {"

Comment (smeeckaert) [inline views/hasmany/items.view.php]: "} elseif (!empty($value) === !false) {" ?

Comment (smeeckaert) [inline views/hasmany/items.view.php]: Done.

**Comment by jguyomard** [comment]
:+1:

File: views/hasmany/items.view.php [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
+    $listItems = array();
```
Comment (Meroje) [inline views/hasmany/items.view.php:7]: Could be moved up while you're at it

**Pull request merged**